packages/trackerless-network/test/unit/TemporaryConnectionRpcLocal.test.ts

```diff
@@ -0,0 +1,39 @@
+import { TemporaryConnectionRpcLocal } from '../../src/logic/temporary-connection/TemporaryConnectionRpcLocal'
+import { MockTransport } from '../utils/mock/Transport'
+import { createMockPeerDescriptor } from '../utils/utils'
+import { ListeningRpcCommunicator, getDhtAddressFromRaw } from '@streamr/dht'
+
+describe('TemporaryConnectionRpcLocal', () => {
+
+    const peerDescriptor = createMockPeerDescriptor()
+    let rpcCommunicator: ListeningRpcCommunicator
+    let rpcLocal: TemporaryConnectionRpcLocal
+
+    beforeEach(() => {
+        rpcCommunicator = new ListeningRpcCommunicator('mock', new MockTransport())
+        rpcLocal = new TemporaryConnectionRpcLocal({
+            localPeerDescriptor: peerDescriptor,
+            rpcCommunicator
+        })
+
+    })
+
+    afterEach(() => {
+        rpcCommunicator.destroy()
+    })
+
+    it('Open Connection', async () => {
+        const caller = createMockPeerDescriptor()
+        await rpcLocal.openConnection(peerDescriptor, { incomingSourceDescriptor: caller } as any)
```

Comment:
Should the 1st argument be instance of `TemporaryConnectionRequest` (de facto: unused)?